src/main/java/stream/flarebot/flarebot/objects/GuildWrapper.java

```diff
@@ -247,4 +249,48 @@ public boolean getBetaAccess() {
         if (tags == null) tags = new ConcurrentHashMap<>();
         return tags;
     }
+
+    public boolean eventEnabled(ModlogEvent event) {
+        checkEventsArray();
+        return enabledEvents.contains(event);
+    }
+
+    public boolean enableEvent(ModlogEvent event) {
+        checkEventsArray();
+        return enabledEvents.add(event);
+    }
+
+    public boolean disableEvent(ModlogEvent event) {
+        checkEventsArray();
+        return enabledEvents.remove(event);
+    }
+
+    public boolean toogleCompactEvent(ModlogEvent event) {
+        if (event.isCompact()) {
+            event.setCompact(false);
+            return false;
+        } else {
+            event.setCompact(true);
+            return true;
+        }
+    }
+
+    public List<ModlogEvent> getEnabledEvents() {
+        checkEventsArray();
+        return enabledEvents;
+    }
+
+    private void checkEventsArray() {
+        if (enabledEvents == null) {
+            enabledEvents = new ArrayList<>();
```

Comment:
Don't hard code these, use an `isDefault` flag on the enum

Reply:
I'd half to hard code then at some point along the line to actually set the defaults. unless i make each event a boolean I don't see how adding a isDefault will help